apps/condo/domains/meter/schema/RegisterMetersReadingsService.js

```diff
@@ -0,0 +1,266 @@
+        },
+    ],
+
+    mutations: [
+        {
+            access: access.canRegisterMetersReadings,
+            schema: 'registerMetersReadings(data: RegisterMetersReadingsInput!): [MeterReading]',
+            resolver: async (parent, /**{ data: RegisterMetersReadingsInput }*/args, context) => {
+                const { data: { dv, sender, organization, readings } } = args
+
+                if (readings.length > READINGS_LIMIT) {
+                    throw new GQLError(ERRORS.TOO_MUCH_READINGS(readings.length), context)
+                }
+
+                /** @type Organization */
+                const organizationData = await getById('Organization', organization.id)
+
+                if (!organizationData || !!organizationData.deletedAt) {
+                    throw new GQLError(ERRORS.ORGANIZATION_NOT_FOUND, context)
+                }
+
+                const locale = extractReqLocale(context.req) || conf.DEFAULT_LOCALE
+
+                const propertyResolver = new PropertyResolver({ context })
+                propertyResolver.tin = organizationData.tin
+
+                const resolvedAddresses = await propertyResolver.normalizeAddresses(readings.reduce((res, reading, index) => ({
+                    ...res,
+                    [reading.address]: {
+                        address: reading.address,
+                        addressMeta: reading.addressMeta,
+                    },
+                }), {}))
+
+                const addressesKeys = readings.map((reading) => get(resolvedAddresses, [reading.address, 'addressResolve', 'propertyAddress', 'addressKey'])).filter(Boolean)
+
+                /** @type Property[] */
+                const properties = await find('Property', {
+                    organization,
+                    deletedAt: null,
+                    addressKey_in: addressesKeys,
+                })
+
+                const resultRows = []
+
+                for (const reading of readings) {
+                    const addressKey = get(resolvedAddresses, [reading.address, 'addressResolve', 'propertyAddress', 'addressKey'])
+
+                    const property = properties.find((p) => p.addressKey === addressKey)
+
+                    if (!property) {
+                        resultRows.push(new GQLError(ERRORS.PROPERTY_NOT_FOUND, context))
+                        continue
+                    }
+
+                    let meterId
+                    const foundMeter = await getByCondition('Meter', {
+                        organization,
+                        property: { id: property.id },
+                        unitType: reading.addressMeta.unitType,
+                        unitName: reading.addressMeta.unitName,
+                        accountNumber: reading.accountNumber,
+                        number: reading.meterNumber,
+                    })
```

Comment:
Maybe add try/catch for this request so that mutation does not fail?

Reply:
It's better to stop processing if we found 2+ meters here. It means we have bad rows in database.

Reply:
That's right. We should write this error into `resultRows`, and not throw this error into mutation. 
This is how you handle other errors in rows, but for some reason not here.
If there is an error in a row, then write the error to `resultRows` and continue on to the next row

Reply:
Ok, the error was added. But it is impossible to test due to `meter_unique_organization_and_number` constraint.